index.js

```diff
@@ -0,0 +1,21 @@
+$(document).delegate('a:Event(click)', 'click', function (e) { e.preventDefault(); });
+
+$( ".readmore" ).click(function() {
+  $( "#show-this-on-click" ).slideDown();
+});
+$( ".readmore" ).click(function() {
+  $( "#hidebtn" ).show();
+});
+$( ".readmore" ).click(function() {
+  $( ".readmore" ).hide();
+});
+
+$( ".readless" ).click(function() {
+  $( "#hidebtn" ).slideUp();
+});
+$( ".readless" ).click(function() {
+  $( "#show-this-on-click" ).hide("slow");
+});
+$( ".readless" ).click(function() {
+  $( ".readmore" ).show(1000);
```

Comment:
I like this, you realised a UX issue and you found a hack for it, but theres a better way of doing this.

```
	$( ".readless" ).click(function() {
	  $( "#show-this-on-click" ).slideUp(function(){
	  	$( "#hidebtn" ).hide();
	  	$( ".readmore" ).show();
	  });
	});
```

Put a function in slideUp and whatever goes into that function is executed after the slideUp action